views/migration_lifter.php

```diff
@@ -133,8 +136,9 @@
 							<div class="tutor-col-md-8 tutor-d-flex tutor-flex-wrap">
 								<sapn class="backup-now-subtile tutor-fs-7"><?php _e( 'Please take a complete a backup for safety.', 'tutor-lms-migration-tool' ); ?></sapn>
 								<form id="tutor_migration_export_form" method="post" enctype="multipart/form-data">
-									<input type="hidden" id="tutor_migration_vendor" name="tutor_migration_vendor" value="lp">
+									<input type="hidden" id="tutor_migration_vendor" name="tutor_migration_vendor" value="lif">
```

Comment:
Sure about it's handling with `lif` instead of `lp`?

Reply:
yes bhai